tsconfig.extend.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
+    "paths": {
+      "@theme/*": [ "./theme/*" ],
```

Comment:
Thank you for going through and updating all the paths! This is awesome. 

I was wondering if it's worth implementing a pattern that allows for new folders to be added without needing to update this config? For example, here: https://dev.to/chgldev/how-to-setup-pretty-import-paths-in-a-create-react-app-application-5dck uses a common alias so that any folder under src is reachable

Reply:
Oh wow this is neato